### ISSUE TITLE
Cop to limit the number of safe navigation calls in a chain

### ISSUE THREAD

**Issue by fatkodima** [rubocop/rubocop#13171]
It is painful to see many `&.` calls in a chain.

If I want to have at most 2 safe navigation calls in a row, for example, then:
```ruby
# bad
foo&.bar&.baz&.zoo

# good
foo&.bar&.baz
```

**Comment by fatkodima** [comment]
Opened https://github.com/rubocop/ruby-style-guide/issues/946
